Gemfile

```diff
@@ -21,6 +21,7 @@ gem "bootsnap", ">= 1.4.4", require: false
 gem "govuk-components"
 gem "govuk_design_system_formbuilder"
 gem "hotwire-rails"
+gem "singleton"
```

Comment:
Worth double checking if this is actually needed or if it's in the standard library now?